### ISSUE TITLE
i cannot find label information used in training model

### ISSUE THREAD

**Issue by SniperTian** [meteahishali/SRL-SOA#2]
In utils.py   line126-line130
     if weights == 'False':
            model.fit(xx, xx, batch_size = batchSize,
                    callbacks=callbacks_osen, shuffle=True,
                    validation_data=(xx, xx), epochs = epochs)
            print(modelType + ' is trained!')
maybe it should be xx,yy not xx,xx (xx = classData['x_train'], yy = classData['y_train'])
I'm not good at deeplearning, thus this 'bug' maybe correct. Can you answer my question? Thank you !

**Comment by meteahishali** [comment]
Hi, this is an autoencoder unsupervised training, so the label is the input itself. Labels are used only for SVM training. I am closing this issue, please feel free to open a new issue, if you like to ask more questions.

Best Regards,
Mete Ahishali
